hack/gke/testrelease-gke.yaml

```diff
@@ -28,7 +28,7 @@ items:
           containers:
             - name: check-oidc
               image: gcr.io/projectsigstore/cosign:v1.5.1
-              args: [ "sign", "--k8s-keychain", "--force", "--fulcio-url", "http://fulcio.fulcio-system.svc", "--rekor-url", "http://rekor.rekor-system.svc", "gcr.io/chainguard-dev/nginx" ]
+              args: [ "sign", "--k8s-keychain", "--force", "--fulcio-url", "http://fulcio.fulcio-system.svc", "--rekor-url", "http://rekor-server.rekor-system.svc", "gcr.io/chainguard-dev/nginx" ]
```

Comment:
It's odd that this has rekor-server.rekor-system.svc instead of rekor.rekor-system.svc

Reply:
its how the helm-template sets it up, since it has rekor-server and rekor-redis. https://github.com/sabre1041/sigstore-charts/blob/scaffold/charts/rekor/values.yaml#L56

Reply:
Ok, makes sense, just wanted to double-check :)